src/amo/components/LandingPage.js

```diff
@@ -108,9 +109,29 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = 'Change your browser’s appearance. Choose from thousands of themes to give Firefox the look you want.';
+    const extensionsText = 'Install powerful tools that make browsing faster and safer, add-ons make your browser yours.';
 
     return (
       <div className={classNames('LandingPage', `LandingPage-${addonType}`)}>
+
+        <div className="Addon-Header">
```

Comment:
Our class names are formatted as `ComponentName-element-name[--state]`. So this should be something like `LandingPage-header`. The other classes will need to be updated as well.

You shouldn't need `--state` in any of these but that would be for something like install switches where it would have a normal class like `InstallSwitch` and then when it is in the installing state (flipped on with animated stripes) you add `InstallSwitch--installing` then once it is installed you remove that and add `InstallSwitch--installed`.